README.md

```diff
@@ -167,8 +167,8 @@ func InitializeScenario(ctx *godog.ScenarioContext) {
 Alternatively, you can also specify the keyword (Given, When, Then...) when creating the step definitions:
 ``` go
 func InitializeScenario(ctx *godog.ScenarioContext) {
-        ctx.Given(`^I eat (\d+)$`, iEat)
```

Comment:
Thank you for your contribution!

I can see that you have reordered the Given and When calls. Actually, the logical order of calls and setting them up was correct - Given, When, Then - so the new order When, Given, Then actually is counter-intuitive.

Can you please revert your change and instead reorder the funcs above? Thx

Reply:
Sure, I will put the Given statement first. The reason I made the change is because in the feature file "Given" is linked to "there are 12 godogs" and "When" is linked to "I eat 5"